app/views/pageds/show.html.erb

```diff
@@ -26,16 +26,20 @@
         <% end %>
       </li>
     </ul>
+    <div>
+      <%= button_to 'Update', edit_paged_path(@paged), :method => :get %>
+    </div>
   </div>
   
   <!-- Cover Image -->
-  <div style="float:left; padding-left:50px;">
-    <%= image_tag(@ordered[0]["ds_url"], height: "64", width: "64" ) %>
+  <div style="float:left; padding-left:50px; text-align:center;">
+    <% if @ordered[0] %>
```

Comment:
We ought to have a functional test for this.  I was surprised by the same issue when manually testing another patch.